gateway/api/schedule.py

```diff
@@ -153,3 +155,41 @@ def get_jobs_to_schedule_fair_share(slots: int) -> List[Job]:
         job_filter |= Q(author=entry["author"]) & Q(created=entry["job_date"])
 
     return Job.objects.filter(job_filter)
+
+
+def check_job_timeout(job: Job, job_status):
```

Comment:
I always get a bit mixed up on how Python handles passing variables, so just want to ask as a sanity check that the updates made to the `job` within these functions are seen outside of these functions, right?

Reply:
That's correct.  I looked it up :-) and also tested.